packages/fields/README.md

```diff
@@ -18,6 +18,10 @@ npm install @wordpress/fields --save
 
 Author field for BasePost.
 
+### BasePost
+
+Undocumented declaration.
```

Comment:
By adding a JSDoc comment to the exported entity this won't be undocumented.

Reply:
Hi @oandregal I added as JSDoc but the problem is persisting. BasePostWithEmbeddedAuthor was already undocumented so I don't think this is a blocker.